test/datatypes/clinical-test.ts

```diff
@@ -119,6 +119,21 @@ describe('Concept', () => {
   });
 });
 
+describe('ValueSet', () => {
```

Comment:
Do we want to title this `CQLValueSet`?

Reply:
Rawr- this is what I get for doing a last minute symbol rename. lol- thanks for catching!